### PR TITLE
Added a section on security

### PR DESCRIPTION
**Pull request by abidlabs**
Closes: #3397

### CODE REVIEW

File: guides/01_getting-started/03_sharing-your-app.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ How to share your Gradio app:
 6. [Adding authentication to the page](#authentication)
 7. [Accessing Network Requests](#accessing-the-network-request-directly)
 8. [Mounting within FastAPI](#mounting-within-another-fastapi-app)
+9. [Security](#security)
```
Comment (aliabd) [inline guides/01_getting-started/03_sharing-your-app.md:13]: this is not a valid link
>`[Security](#security)`

`[Security](#security-and-file-access)`

File: guides/01_getting-started/03_sharing-your-app.md [hunk 2]
```diff
@@ -157,3 +160,17 @@ Here's a complete example:
...
 
 Note that this approach also allows you run your Gradio apps on custom paths (`http://localhost:8000/gradio` in the example above).
+
+## Security and File Access
+
+Sharing your Gradio app with others (by hosting it on Spaces, on your own server, or through temporary share links) **exposes** certain files on the host machine to users of your Gradio app. This is done so that Gradio apps are able to display output files created by Gradio or created by your prediction function.
+
+In particular, Gradio apps grant users access to three kinds of files:
+
+* files in the same folder (or a subdirectory) of where the Gradio script is launched from. For example, if the path to your gradio scripts is `/home/usr/scripts/project/app.py` and you launch it from `/home/usr/scripts/project/`, then users of your shared Gradio app will be able to access any files inside `/home/usr/scripts/project/`. This is needed so that you can easily reference these files in your Gradio app.
```
Comment (aliabd) [inline guides/01_getting-started/03_sharing-your-app.md:170]: capitalize first word in these bullet points

**Review by aliabd — APPROVED** [review]
LGTM with minor changes

**Comment by abidlabs** [comment]
Thank you @aliabd! Agree with the changes

**Pull request merged**